apps/code-review/src/review-request/review-request.controller.ts

```diff
@@ -0,0 +1,89 @@
+import { AuthorizedUser } from '@acua/shared/user/interfaces';
+import {
+    Body,
+    Controller,
+    Get,
+    HttpStatus,
+    Param,
+    Patch,
+    Post,
+    Request,
+    UseGuards
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { ApiBearerAuth, ApiOperation, ApiParam, ApiResponse, ApiTags } from '@nestjs/swagger';
+import { CodeReviewRequestStatusEnum, RouteParamEnum } from './enums';
+import { CodeReviewDataRequestDto, CodeReviewRequestDto } from './models';
+import { CodeReviewRequestDocument } from './schemas';
+import { ReviewRequestService } from './services';
+
+@ApiBearerAuth()
+@ApiTags('review-request')
+@UseGuards(AuthGuard('jwt'))
+@Controller('review-request')
+export class ReviewRequestController {
+    constructor(private readonly reviewRequestService: ReviewRequestService) {}
+
+    @ApiResponse({
+        description: `Returns list of code review requests`,
+        status: HttpStatus.OK,
+        type: [CodeReviewRequestDto]
+    })
+    @ApiOperation({ summary: 'Get list of code review requests' })
+    @Get()
+    public find(): Promise<CodeReviewRequestDto[]> {
+        return this.reviewRequestService.get();
+    }
+
+    @ApiResponse({
+        description: `Returns code review request`,
+        status: HttpStatus.OK,
+        type: CodeReviewRequestDto
+    })
+    @ApiParam({
+        name: 'id',
+        description: 'Specifies which code review request to retrieve',
+        type: 'string'
+    })
+    @ApiOperation({ summary: 'Get particular code review request by specifying its id as param' })
+    @Get(`:${RouteParamEnum.Id}`)
+    public findOne(@Param() params: any): Promise<CodeReviewRequestDto> {
+        return this.reviewRequestService.getOne(params[RouteParamEnum.Id]);
+    }
+
+    @ApiResponse({
+        description: `Returns raw code review request`,
+        status: HttpStatus.CREATED
+    })
+    @ApiOperation({ summary: 'Create a new code review request' })
+    @Post()
+    public create(
+        @Request() req: any,
```

Comment:
```suggestion
        @Request() req: Request,
```

from express package

Reply:
В обʼєкті Request немає поля з даними payload юзера.
Окрім як створити окремий інтерфейс і розширити його Request - не бачу, як можна отримати юзера.